sdk/communication/azure-communication-chat/tests/test_chat_thread_client_e2e_async.py

```diff
@@ -304,15 +314,17 @@ async def test_list_read_receipts(self):
             await self._create_thread()
 
             async with self.chat_thread_client:
-                await self._send_message()
 
-                # send read receipts first
-                await self.chat_thread_client.send_read_receipt(self.message_id)
-                if self.is_live:
-                    await asyncio.sleep(2)
+                for i in range(2):
+                    await self._send_message()
+
+                    # send read receipts first
+                    await self.chat_thread_client.send_read_receipt(self.message_id)
+                    if self.is_live:
+                        await asyncio.sleep(2)
```

Comment:
As Tural mentioned, we need to avoid using sleep

Reply:
time.sleep() has been moved inside a single helper function now